runtime/doc/vim_diff.txt

```diff
@@ -809,6 +809,13 @@ Providers:
 - |if_pyth|: *python-bindeval* *python-Function* are not supported.
 - *if_tcl*
 
+Scripts:
```

Comment:
these items can live in the Plugins: section above. it really doesn't matter that they aren't technically plugins.